libs/cardano-ledger-api/src/Cardano/Ledger/Api/Tx/Address.hs

```diff
@@ -1,13 +1,27 @@
 module Cardano.Ledger.Api.Tx.Address (
   -- * Address
   Addr (..),
+  getNetwork,
   BootstrapAddress (..),
   serialiseAddr,
+
+  -- ** Strict decoders
+
+  -- | Decoders below will only decode addresses that are allowed to be placed on chain
+  -- today. Historically there were a few bugs in the decoder which allowed a few
+  -- malformed addressed to be placed on chain. If you need backwards compatibility, reach
+  -- out for `decodeAddrLenient`.
   decodeAddr,
   decodeAddrEither,
   decodeAddrShort,
   decodeAddrShortEither,
-  getNetwork,
+
+  -- ** Lenient decoders
+
+  -- | These a lenient decoders that do not fail for addresses with known bugs
```

Comment:
Maybe: `These *are* lenient decoders .. `   or `These lenient decoders do not fail ..`

Reply:
Done. Thank you.